Ryujinx.Ava/Ryujinx.Ava.csproj

```
@@ -66,6 +66,13 @@
       <TargetPath>THIRDPARTY.md</TargetPath>
     </ContentWithTargetPath>
   </ItemGroup>
+   
+   <ItemGroup>
+     <ContentWithTargetPath Include="..\distribution\linux\start.sh" Condition="'$(RuntimeIdentifier)' != 'win10-x64' AND '$(RuntimeIdentifier)' != 'osx-x64'">
```

```suggestion
     <ContentWithTargetPath Include="..\distribution\linux\Ryujinx.sh" Condition="'$(RuntimeIdentifier)' != 'win10-x64' AND '$(RuntimeIdentifier)' != 'osx-x64'">
```
How about renaming this to `Ryujinx.sh`?

---

Done